openslides/motions/apps.py

```diff
@@ -14,19 +14,40 @@ class MotionsAppConfig(AppConfig):
     def ready(self):
         # Import all required stuff.
         from openslides.core.config import config
-        from openslides.core.signals import permission_change, user_data_required
+        from openslides.core.signals import (
+            permission_change,
+            post_permission_creation,
+            user_data_required,
+        )
         from openslides.utils.rest_api import router
         from .config_variables import get_config_variables
         from .projector import get_projector_elements
-        from .signals import create_builtin_workflows, get_permission_change_data, required_users
-        from .views import CategoryViewSet, MotionViewSet, MotionBlockViewSet, MotionPollViewSet, MotionChangeRecommendationViewSet, WorkflowViewSet
+        from .signals import (
+            create_builtin_workflows,
+            delete_old_comment_permission,
+            get_permission_change_data,
+            required_users,
+        )
+        from .views import (
+            CategoryViewSet,
+            MotionViewSet,
+            MotionBlockViewSet,
+            MotionPollViewSet,
+            MotionChangeRecommendationViewSet,
+            WorkflowViewSet,
+        )
 
         # Define config variables and projector elements.
         config.update_config_variables(get_config_variables())
         register_projector_elements(get_projector_elements())
 
         # Connect signals.
-        post_migrate.connect(create_builtin_workflows, dispatch_uid='motion_create_builtin_workflows')
+        post_migrate.connect(
+            create_builtin_workflows,
+            dispatch_uid='motion_create_builtin_workflows')
+        post_permission_creation.connect(
+            delete_old_comment_permission,
```

Comment:
This function has nothing to do with appstart but with migration process. So I prefer not to pollute the code here with this stuff ...

Reply:
Done.